### ISSUE TITLE
ISO 19115-1 association with ISO 19165-1:2018

### ISSUE THREAD

**Issue by ejbleys** [ISO-TC211/StandardsTracker#508]
The UML for ISO 19165-1:2018 identifies associations form ISO 19115-1:2014 classes to ISI 19165-1 classes
When ISO 19115-1 is reviewed next it would be worthwhile incorporating the 
     [0..1] association from DS_Series (ISO 19115-1:2014) to GP_BornDigital ISO 19165-1
     [0..*] association from DS_Dataset (ISO 19115-1:2014) to GP_ScannedDigitalCopy ISO 19165-1
Alternatively the classes in "Content Information" in the ISO 19156-1 UML could be included in an updated version of ISO 19115-1
     GP_BornDigital; GP_ScannedDigitalCopy; <<abstract>> GP_Digital; GP_MasterFile; and GP_DistributionFile
The "Content Information" in the ISO 19156-1 UML has currently been included in the draft XML schemas using assumed role names (will eventually appear at https://schemas.isotc211.org/19165/-1/gpm/1.0.0/geospatialPreservationMetadata.xsd)

**Comment by jetgeo** [comment]
Removed HMMG label, as this will be handled by the 19115 revision

**Comment by PeterParslow** [comment]
> The UML for ISO 19165-1:2018 identifies associations form ISO 19115-1:2014 classes to ISI 19165-1 classes

@ejbleys - could you highlight where you see these associations? In the current UML model, the "preservation model structure" diagram includes associations between DS_Series & GP_BornDigital, and between DS_Dataset and both GP_BornDigital and GP_ScannedDigitalCopy. I don't believe this diagram appears in the standard (it doesn't have a figure number). These associations don't have any navigability; I think that defaults to both ends being navigable - but if I inspect the classes themselves (rather than relying on the diagram), these associations are described as "outgoing", which isn't a problem.  I do need to get access to the 19165-1 document (I should have that access through BSI, just not working this morning!)

The associations shown in Figure 3 had a single navigable end each, and I think that implies that the other end isn't navigable - so no problem there.

Are there other associations I haven't spotted, which do appear in the standard document?

In other words, is there a problem in the published standard, "just" in the harmonized model, or perhaps just in the unpublished diagram, not in the actual classes themselves?

The separate question of whether GP_BornDigital and GP_ScannedDigitalCopy are of enough general use to be adopted into a new edition of ISO 19115-1 will be considered by the revision project.

After those discussions, we'll know whether we need to:

- revise ISO 19165-1, which won't fix any issues in the current schema
- fix "hidden" parts of the model & then regenerate / fix the schema

**Comment by PeterParslow** [comment]
Ah. Digging further in the current model, it claims two (!) incoming associations from LI_Lineage to GP_PreservationMetadata. One of those appears in a diagram in the model which is labelled Figure C.2, in which is an aggregation that's indicated navigability is _from_ GP_PreservationMetadata (and similarly from GP_AssociatedResource to LI_ProcessStep

That suggests (to me!) that these associations do not appear in the published standard and perhaps should "simply" have their direction fixed to match their navigability. If that's the case, then after that regenerating the 19165-1 schemas would (I think) remove the problem.

**Comment by PeterParslow** [comment]
Peter's dig was into the wrong part (19165-2) & the "issue" there is not an issue (navigability OK) - close that sub-thread

Regarding the original issue, in ISO 19165-1 the diagram "Preservation Model Structure" doesn't appear in the document and that document has no mention of the two classes. Refer back to WG1

**Comment by 6footdestiny** [comment]
As a member of the orginal working group in the very early stages of this work from memory I dont recall those classes ing  discussed. It's possible they were discussed at the much later revision stage (which I wasnt involved in ) but from lurking on the sidelines I don't recall any explicit mention of them so it's likely to be an accidental omission rather than a deliberate one IMO.
